com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -87,6 +87,10 @@ private RandomAccessFile createRandomAccessFile(File handle) throws IOException
 
     @Override
     public Task<Void> create(IResource input) throws IOException, CompileExceptionError {
+        boolean nonStandardGameProjectFile = !project.getGameProjectResource().getAbsPath().equals(input.getAbsPath());
```

Comment:
I initially tested throwing an exception here if another *.project file was detected but this messes with the tests we have in `engine/engine/src/test`. The tests contain multiple project files in subfolders and adjusting the tests to no longer rely on this would require quite a bit of work.